src/commands/snapshot.js

```diff
@@ -1,7 +1,72 @@
+const { propEq, filter, head } = require('ramda')
+const NONE = 'None'
+const DO_NOTHING = 'Nothing to do ¯\\_(ツ)_/¯'
+
+const createSolidarityFile = async (context) => {
+  const { print, printSeparator } = context
+  // list visible plugins
+  printSeparator()
+  print.info('Available technology plugins:\n')
+  if (context.pluginsList.length > 0) {
+    const pluginOptions = [NONE]
+    context.pluginsList.map((plugin) => {
+      print.info(`   ${plugin.name}:\t ${plugin.description}`)
```

Comment:
Sadly this tab won’t align things that have > 4 character difference. Or at least that’s how I understand tabs to work in the terminal.

Reply:
Damn!  I was trying to use table print, but it needed such a specific structure.  I'll get back to formatting on the final layer of polish.